native/src/modules/categories/components/CategoryList.js

```diff
@@ -54,6 +58,17 @@ const CategoryThumbnail = styled(Image)`
  * Displays a ContentList which is a list of categories, a caption and a thumbnail
  */
 class CategoryList extends React.Component<PropsType> {
+  getCachedThumbnail (category: CategoryModel): ?string {
```

Comment:
Why did you have to add this? is this used anywhere?

Reply:
I think I just forgot to remove it again.